client/core/core.go

```diff
@@ -593,6 +594,20 @@ func (c *Core) CreateWallet(appPW, walletPW string, form *WalletForm) error {
 // loadWallet uses the data from the database to construct a new exchange
 // wallet. The returned wallet is running but not connected.
 func (c *Core) loadWallet(dbWallet *db.Wallet) (*xcWallet, error) {
+	// todo: wallet config file should be parsed when wallet is being created
+	// and the parsed options save to db. Remember to use the default ini path
+	// if an emtpy path is passed to CreateWallet()
+	if dbWallet.INIPath == "" {
```

Comment:
I know this is all changing soon anyway, but I would move the empty-to-default conversion to `CreateWallet`, and actually error here if `INIPath` is empty.